app/services/update_chat_channel_membership_service.rb

```diff
@@ -0,0 +1,15 @@
+class UpdateChatChannelMembershipService < ApplicationService
+  def initialize(chat_channel, membership, params)
+    @chat_channel = chat_channel
+    @membership = membership
+    @params = params
+  end
+
+  attr_accessor :chat_channel, :membership, :params
+
+  # @TODO: Add dynamic content
+  def perform
+    membership.update(params)
```

Comment:
Same comment as above, since we are simply doing an update statement here it doesnt feel like it warrants the added complexity of an entire service.